### DISCUSSION TITLE
Mender for scarthgap-l4t-r35.x

### DISCUSSION THREAD

**Discussion by tjhgit** [OE4T/tegra-demo-distro#311] [Q&A] [answered]
Dear all,

somehow the meta-mender layer is missing for L4T 35.x. Where can I find it? Or can I somehow easily adapt the kirkstone meta-mender layer to work for this newer release?

**Comment by dwalkes** [comment, marked as answer]
Mender was removed from the demo distro in 35.x, but you can find it at https://github.com/mendersoftware/meta-mender-community
